index.html

```diff
@@ -158,27 +178,28 @@ <h3 class="card-main-title">Profesional Art Printing Data</h3>
                                <li><button type="button" class="btn-card">Bootstrap</button></li>
                                <li><button type="button" class="btn-card">Ruby</button></li>
                             </ul>
-                        </section>
-                            
+                        </section>  
                         </section> 
                         <button type="button" class="card-big-btn" >See project</button>    
                     </section>
                     
             </section> 
         </article>  
         <article id="work-experience">
-            <section class="background-image">
             <section class="About-me">
+               <section class="About-content ">
                <h3 class="about-title">About me</h3>
                <p class="about-description">Hello I’m a software developer! I can help you build a product , 
                feature or website Look through some of my work and experience!
                If you like what you see and have a project you need coded, 
                don’t hesitate to contact me.
                </p>
                <button type="button" class="get-my-resume">Get my resume</button>
-               <section class="line"></section> 
-            </section> 
+            </section>
```

Comment:
- [ ] I think the `about me` section also needs an improvement. 🪐   On screen sizes between 900px and 1230px your background image in this section contrasts with the text on your website, This makes it difficult to read the text. Please consider fixing this. -: U can try reducing the background image size at this screen size 👍
- Screenshot:
![Screenshot 2023-04-01 094635](https://user-images.githubusercontent.com/107632780/229277276-128e271f-3ada-4492-b43a-d42500b10bd0.png)